src/indexer/services/evmIndexer/evmIndexer.ts

```diff
@@ -20,8 +20,8 @@ import { OfacComplianceService } from "../ofac"
 import { getLogs } from "./evmfilter"
 import { decodeLogs } from "./evmEventParser"
 
-const BLOCK_TIME = 15000
-
+const BLOCK_TIME = Number(process.env.BLOCK_TIME) || 15000
+const REORG_COUNT = Number(process.env.REORG_COUNT) || 10
```

Comment:
@tcar121293 let us just call it something like `BLOCK_DELAY`, as for me `REORG_COUNT` can be misleading